app/app/RegisterScreen.tsx

```diff
@@ -0,0 +1,145 @@
+
+  const onSignUpPressed = () => {
+    // const nameError = nameValidator(name.value)
+    // const emailError = emailValidator(email.value)
+    // const passwordError = passwordValidator(password.value)
+    // if (emailError || passwordError || nameError) {
+    //   setName({ ...name, error: nameError })
+    //   setEmail({ ...email, error: emailError })
+    //   setPassword({ ...password, error: passwordError })
+    //   return
+    // }
+    // navigation.reset({
+    //   index: 0,
+    //   routes: [{ name: 'Dashboard' }],
+    // })
+    // TODO: maybe would be cool if we changed the onChangeText to call a method to query the api to see if a username is available
+    // and then can use HelperText to display if its available or not
+    // TODO: HelperText to tell user if there is a specific problem with the password they are trying to use
+    // TODO: verify both entered passwords are the same
+  }
+
+  return (
+    <View style={styles.bigContainer}>
+      <View style={styles.margins}>
+        {/* <BackButton goBack={navigation.goBack} />
+      <Logo />
+      <Header>Create Account</Header> */}
+        <View style={styles.container}>
+          <UploadImage />
+        </View>
+        <View style={styles.inputContainer}>
+          <Text>Create Account</Text>
+          <TextInput
+            label="Username"
+            returnKeyType="next"
+            value={username.value}
+            onChangeText={(text) => setUsername({ value: text, error: '' })}
+            error={!!username.error}
+          // errorText={username.error}
+          />
+          <TextInput
+            label="Email"
+            returnKeyType="next"
+            value={email.value}
+            onChangeText={(text) => setEmail({ value: text, error: '' })}
+            error={!!email.error}
+            // errorText={email.error}
+            autoCapitalize="none"
+            autoComplete="email"
+            textContentType="emailAddress"
+            keyboardType="email-address"
+          />
+          <TextInput
+            label="Password"
+            returnKeyType="done"
+            value={password.value}
+            onChangeText={(text) => setPassword({ value: text, error: '' })}
+            error={!!password.error}
+            // errorText={password.error}
+            secureTextEntry
+          />
+          <TextInput
+            label="Re-enter Password"
+            returnKeyType="done"
+            value={passwordConfirmation.value}
+            onChangeText={(text) => setPasswordConfirmation({ value: text, error: '' })}
+            error={!!passwordConfirmation.error}
+            // errorText={passwordConfirmation.error}
+            secureTextEntry
+          />
+          <Button
+            mode="contained"
+            onPress={onSignUpPressed}
+            buttonColor='#0d99ff' 
+            textColor='white'
+          >
+            Create Account
```

Comment:
I think you won't need a create account page lol. We can customize our own login page on entra id b2c flow